2023-03-30 19:09:11,840 INFO    MainThread:5156 [wandb_setup.py:_flush():68] Configure stats pid to 5156
2023-03-30 19:09:11,841 INFO    MainThread:5156 [wandb_setup.py:_flush():68] Loading settings from /home/yx/.config/wandb/settings
2023-03-30 19:09:11,841 INFO    MainThread:5156 [wandb_setup.py:_flush():68] Loading settings from /mnt/c/kust/kust/python_code/python_code/code/experiments/unet/wandb/settings
2023-03-30 19:09:11,841 INFO    MainThread:5156 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2023-03-30 19:09:11,841 INFO    MainThread:5156 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program_relpath': 'myunet.py', 'program': '/mnt/c/kust/kust/python_code/python_code/code/experiments/unet/myunet.py'}
2023-03-30 19:09:11,852 INFO    MainThread:5156 [wandb_init.py:_log_setup():476] Logging user logs to ./runs/wandb/run-20230330_190911-39oesk1o/logs/debug.log
2023-03-30 19:09:11,852 INFO    MainThread:5156 [wandb_init.py:_log_setup():477] Logging internal logs to ./runs/wandb/run-20230330_190911-39oesk1o/logs/debug-internal.log
2023-03-30 19:09:11,852 INFO    MainThread:5156 [wandb_init.py:init():516] calling init triggers
2023-03-30 19:09:11,852 INFO    MainThread:5156 [wandb_init.py:init():519] wandb.init called with sweep_config: {}
config: {}
2023-03-30 19:09:11,852 INFO    MainThread:5156 [wandb_init.py:init():569] starting backend
2023-03-30 19:09:11,853 INFO    MainThread:5156 [wandb_init.py:init():573] setting up manager
2023-03-30 19:09:11,856 INFO    MainThread:5156 [backend.py:_multiprocessing_setup():102] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-03-30 19:09:11,859 INFO    MainThread:5156 [wandb_init.py:init():580] backend started and connected
2023-03-30 19:09:11,861 INFO    MainThread:5156 [wandb_init.py:init():658] updated telemetry
2023-03-30 19:09:11,872 INFO    MainThread:5156 [wandb_init.py:init():693] communicating run to backend with 60 second timeout
2023-03-30 19:10:11,973 ERROR   MainThread:5156 [wandb_init.py:init():704] backend process timed out
2023-03-30 19:10:11,977 ERROR   MainThread:5156 [wandb_init.py:init():713] encountered error: Error communicating with wandb process
For more info see: https://docs.wandb.ai/library/init#init-start-error
2023-03-30 20:49:28,168 WARNING MsgRouterThr:5156 [router.py:message_loop():77] message_loop has been closed
